### ISSUE TITLE
[🐛 Bug]: Edge Driver returning 'Cannot determine loading status' for specific URL

### ISSUE THREAD

**Issue by jgilbrook** [SeleniumHQ/selenium#10893]
### What happened?

When using the Edge driver to go on IBM FixCentral website, it returns the attached error stack. When tested with www.google.com and www.cnn.com, it works with no problem. The only URL I have noticed this issue with is this specific one:
https://www.ibm.com/support/fixcentral/swg/identifyFixes?query.parent=IBM%20Security&query.product=ibm~Other%20software~IBM%20Security%20QRadar%20SIEM&query.release=7.40&query.platform=Linux

When used with the Chrome driver, it faces no problems.

I have verified my Edge browser is running the latest version, has the correct driver, and my selenium version is good. 

### How can we reproduce the issue?

```shell
import os.path 
import re
 import tarfile
 from tkinter import * 
from tkinter.filedialog import askopenfilename 

fixCentral_URL = 'https://www.ibm.com/support/fixcentral/swg/identifyFixesquery.parent=IBM%20Security&query.product=ibm~Other%20software~IBM%20Security%20QRadar%20SIEM&query.release='+ matchesToStr + '.0&query.platform=Linux' 

from selenium import webdriver
 import time 
driver = webdriver.Edge('./msedgedriver') 
driver.get(fixCentral_URL)
driver.find_element_by_xpath('/html/body/div[1]/div[4]/main/div/div/div/div/div[1]/div[1]/form/p[2]/button[1]').click() 
```

### Relevant log output

```shell
selenium.common.exceptions.WebDriverException: Message: unknown error: cannot determine loading status
from unknown error: unexpected command response
  (Session info: MicrosoftEdge=103.0.1264.62)
Stacktrace:
Backtrace:
	Microsoft::Applications::Events::EventProperties::unpack [0x00007FF6D8E66312+24642]
	Microsoft::Applications::Events::GUID_t::GUID_t [0x00007FF6D8DB9A62+334098]
	Ordinal0 [0x00007FF6D8957915+620821]
	Ordinal0 [0x00007FF6D89464C5+550085]
	Ordinal0 [0x00007FF6D8945A7B+547451]
	Ordinal0 [0x00007FF6D8944E45+544325]
	Ordinal0 [0x00007FF6D8943BB0+539568]
	Ordinal0 [0x00007FF6D8944133+540979]
	Ordinal0 [0x00007FF6D894FE00+589312]
	Ordinal0 [0x00007FF6D8960492+656530]
	Ordinal0 [0x00007FF6D8964AA9+674473]
	Ordinal0 [0x00007FF6D89444D5+541909]
	Ordinal0 [0x00007FF6D8960051+655441]
	Ordinal0 [0x00007FF6D89C3516+1062166]
	Ordinal0 [0x00007FF6D89AECB3+978099]
	Ordinal0 [0x00007FF6D8985F00+810752]
	Ordinal0 [0x00007FF6D8987508+816392]
	Microsoft::Applications::Events::EventProperty::EventProperty [0x00007FF6D8C1D578+135480]
	Microsoft::Applications::Events::EventProperty::EventProperty [0x00007FF6D8C04C77+34871]
	Microsoft::Applications::Events::EventProperty::EventProperty [0x00007FF6D8C07CFC+47292]
	Microsoft::Applications::Events::EventProperty::to_string [0x00007FF6D8A611B6+23222]
	Microsoft::Applications::Events::GUID_t::GUID_t [0x00007FF6D8DC0F6B+364059]
	Microsoft::Applications::Events::GUID_t::GUID_t [0x00007FF6D8DC59A4+383060]
	Microsoft::Applications::Events::GUID_t::GUID_t [0x00007FF6D8DC5AFD+383405]
	Microsoft::Applications::Events::time_ticks_t::time_ticks_t [0x00007FF6D8DD040E+39918]
	BaseThreadInitThunk [0x00007FFDA40813D2+34]
	RtlUserThreadStart [0x00007FFDA68954F4+52]
```

### Operating System

Windows Server 2016, Windows 10, Windows 8, Mac OSX 12.4

### Selenium version

Python 4.2.0

### What are the browser(s) and version(s) where you see this issue?

Edge v 103.0.1264.62

### What are the browser driver(s) and version(s) where you see this issue?

MSEdgeDriver 103.0.1264.62

**Comment by jgilbrook** [comment]
Please let me know if you need any clarification, or if you need me to upload the entire code file. It is only ~200 or so lines.

**Comment by titusfortner** [comment]
This was a bug in the driver. 
Fixed in chromedriver v103.0.5060.134 (https://chromedriver.storage.googleapis.com/103.0.5060.134/notes.txt)

msedgedriver is based on chromedriver, but their files don't have release notes or indicate which version they are based on. Try the latest for v103 and see if it works — https://developer.microsoft.com/en-us/microsoft-edge/tools/webdriver/
Otherwise need to wait for a future version to include the fix.
